src/web-rtc-client.js

```diff
@@ -684,17 +661,21 @@ export default class WebRTCClient extends Emitter {
       log: this.config.log || { builtinEnabled: false },
       password: this.config.password,
       uri: `${this.config.authorizationUser || ''}@${this.config.host}`,
+      userAgentString: this.config.userAgentString || 'wazo-sdk',
       transportOptions: {
+        maxReconnectionAttempts: 30,
+        reconnectionTimeout: 2,
         traceSip: false,
         wsServers: `wss://${this.config.host}:${this.config.port || 443}/api/asterisk/ws`,
       },
       sessionDescriptionHandlerFactoryOptions: {
+        alwaysAcquireMediaFirst: true,
```

Comment:
Probably will be better to use it only with firefox?

Reply:
Done